src/neo/SmartContract/Native/Designate/DesignateContract.cs

```diff
@@ -28,16 +29,22 @@ internal override void Initialize(ApplicationEngine engine)
         {
             foreach (byte role in Enum.GetValues(typeof(Role)))
             {
-                engine.Snapshot.Storages.Add(CreateStorageKey(role), new StorageItem(new NodeList()));
+                engine.Snapshot.Storages.Add(CreateStorageKey(role).AddBigEndian(0u), new StorageItem(new NodeList()));
             }
         }
 
         [ContractMethod(0_01000000, CallFlags.AllowStates)]
-        public ECPoint[] GetDesignatedByRole(StoreView snapshot, Role role)
+        public ECPoint[] GetDesignatedByRole(StoreView snapshot, Role role, uint index = uint.MaxValue)
```

Comment:
Currently, smart contracts do not support default parameters. So this may mislead the smart contract to think that only two parameters can be provided.

Reply:
Done